templates/errors/500-confirmation-email.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'layouts/_base.html' %}
+
+{% set page_title = _("Sorry, there was a problem sending the confirmation email") %}
+
+{% block main %}
+<h1>{{ _("Sorry, there was a problem sending the confirmation email") }}</h1>
+<p data-qa="retry">{{ _("You can try to <a href='{retry_url}'>send the email again</a>.".format(retry_url=url_for(request.endpoint))) }}</p>
+<p>{{ _("If this problem keeps happening, please  <a href='{contact_us_url}'>contact us</a> for help.").format(contact_us_url=contact_us_url) }}</p>
+{% endblock %}
```

Comment:
There is duplication between this and the individual response 500 error page. We should consider using one template and passing content from the error handler (as we do for the 429 error page now).

Reply:
I have rationalised this as much as I can however, the template is slightly different currently as IR template has 1 extra paragraph line which I have asked Laura to look at rationalising as I don't think that need to be different.